src/popup/components/Modals/PrivateKeyImport.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <Modal
+    class="private-key-import"
+    from-bottom
+    has-close-button
+    @close="resolve"
+  >
+    <div class="text-center">
+      <p
+        class="text-heading-4"
+        v-text="$t('modals.privateKeyImport.title', {
+          protocol: getProtocolName(protocol),
+        })"
+      />
+      <p v-text="$t('modals.privateKeyImport.subtitle')" />
+    </div>
+
+    <FormTextarea
+      v-model="privateKey"
+      size="sm"
+      :label="$t('modals.privateKeyImport.privateKey')"
+      :placeholder="$t('modals.privateKeyImport.placeholder')"
+      :message="error"
+      :resizable="false"
+      data-cy="field-private-key"
+      enter-submit
+      show-help
+      @submit="importPrivateKey()"
+      @help="showPrivateKeyHelp()"
+    />
+
+    <template #footer>
+      <BtnMain
+        :disabled="!privateKey || !!error"
+        :text="$t('modals.privateKeyImport.btnText')"
+        data-cy="btn-import"
+        @click="importPrivateKey()"
+      />
+    </template>
+  </Modal>
+</template>
+
+<script lang="ts">
+import {
+  defineComponent,
+  PropType,
+  ref,
+  watch,
+} from 'vue';
+import { TranslateResult, useI18n } from 'vue-i18n';
+
+import type { Protocol, ResolveCallback } from '@/types';
+
+import { ProtocolAdapterFactory } from '@/lib/ProtocolAdapterFactory';
+import { ACCOUNT_TYPES, MODAL_HELP } from '@/constants';
+import { useAccounts, useModals } from '@/composables';
+
+import BtnMain from '@/popup/components/buttons/BtnMain.vue';
+import FormTextarea from '@/popup/components/form/FormTextarea.vue';
+import Modal from '@/popup/components/Modal.vue';
+
+export default defineComponent({
+  components: {
+    BtnMain,
+    Modal,
+    FormTextarea,
+  },
+  props: {
+    resolve: { type: Function as PropType<ResolveCallback>, required: true },
+    protocol: { type: String as PropType<Protocol>, required: true },
+  },
+  setup(props) {
+    const { t } = useI18n();
+
+    const { accounts, addPrivateKeyAccount, setActiveAccountByGlobalIdx } = useAccounts();
+    const { openModal } = useModals();
+
+    const privateKey = ref('');
+    const error = ref<TranslateResult>('');
```

Comment:
We don't need to use `TranslateResult` type anymore. `string` is OK since some previous `i18n` version.
```suggestion
    const error = ref('');
```